FRW/Resource/Assets.xcassets/wallet/icon-move-arrow.imageset/Contents.json

```
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "icon-move-arrow.svg",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
```

```suggestion
  "images" : [
    {
      "filename" : "icon-move-arrow.svg",
      "idiom" : "universal"
    }
  ],
```

I think if you forgot to change to `Single Scale` here in the .xcassets.  I'm not sure this will scale properly.

---

yes, you are right. Let me update it in following PR